src/main/kotlin/lotto/domin/LottoMachine.kt

```diff
@@ -0,0 +1,37 @@
+package lotto.domin
+
+import lotto.dto.InputPaymentRequestDto
+import lotto.util.LottoNumberGenerator
+
+class LottoMachine(
+    dto: InputPaymentRequestDto,
+    private val lottoNumberGenerator: LottoNumberGenerator,
```

Comment:
```suggestion
    lottoNumberGenerator: LottoNumberGenerator,
```
실제로 lottoNumberGenerator는 get과 set을 하지 않기에, 프로퍼티로 정의하지 않아도 될 것 같아요.